starter/src/main/jni/starter.c

```diff
@@ -0,0 +1,110 @@
+#include <jni.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/poll.h>
+#include <sys/un.h>
+#include <errno.h>
+#include <string.h>
+#include <fcntl.h>
+
+#define unused __attribute__((__unused__))
+
+#define PREFIX "/data/data/com.termux/files/usr"
```

Comment:
It would also be better to use user-defined `$PREFIX`.. that would add support for almost every third party apps out there...

Reply:
I can pass socket's path from Java code, that is not a problem.